examples/karaf-websocket-example/src/main/java/org/apache/karaf/examples/websocket/WebsocketExampleServlet.java

```diff
@@ -16,16 +16,16 @@
  */
 package org.apache.karaf.examples.websocket;
 
-import org.eclipse.jetty.websocket.servlet.WebSocketServlet;
-import org.eclipse.jetty.websocket.servlet.WebSocketServletFactory;
+import org.eclipse.jetty.websocket.server.JettyWebSocketServlet;
+import org.eclipse.jetty.websocket.server.JettyWebSocketServletFactory;
 
 import javax.servlet.annotation.WebServlet;
 
 @WebServlet(name = "Example WebSocket Servlet", urlPatterns = { "/example-websocket "})
-public class WebsocketExampleServlet extends WebSocketServlet {
+public class WebsocketExampleServlet extends JettyWebSocketServlet {
```

Comment:
We should use the websocket handling here, not the Jetty one.

Reply:
> We should use the websocket handling here, not the Jetty one.

This is just the Jetty 9.4  -> 10 renaming. Do you mean to switch the example to just use JSR356?